Source/WebCore/page/ViewportConfiguration.cpp

```diff
@@ -415,6 +415,21 @@ ViewportConfiguration::Parameters ViewportConfiguration::nativeWebpageParameters
     return parameters;
 }
 
+ViewportConfiguration::Parameters ViewportConfiguration::pluginParameters()
+{
+    Parameters parameters;
+    parameters.width = ViewportArguments::ValueDeviceWidth;
+    parameters.widthIsSet = true;
+    parameters.allowsUserScaling = true;
+    parameters.allowsShrinkToFit = false;
+    parameters.minimumScale = 1;
+    parameters.maximumScale = 1;
```

Comment:
Hm.. `allowsUserScaling=true` alongside `min/maximumScale=1` seems strange. I suppose this is because we want the user to be able to pinch zoom the scroll view, but then intercept any subsequent scaling and handle it purely within the plugin?

Reply:
Yeah this was carried over form the native web page configuration, but after taking a look at how its used I agree that it's a bit strange and we *probably* don't want it with the behavior we intend to implement for the plugin